packages/lit-dev-content/site/tutorials/content/svg-templates/05.md

```diff
@@ -0,0 +1,119 @@
+### Overview
+
+This is chapter of the demo covers the basics of CSS in SVG including
+how to:
+
+- use CSS to style SVG
+- use CSS Custom Properties to create a theme
+
+### Learn
+
+CSS can apply attributes to SVG elements in the same manner CSS
+applies attributes to HTML.
+
+However, unlike HTML, only [SVG Presentation Attributes](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/SVG_and_CSS)
+can be applied through CSS.
+
+It may feel odd but syntax like the example below is valid CSS for SVG
+documents. In the following example, all `<rect>` elements will be as
+tall and wide as their parent element.
+
+```TS
+const helloSvgCss = css`
+	rect {
+		x: 0;
+		y: 0;
+		width: 100%;
+		height: 100%;
+	}
+`;
+```
+
+Since SVG can be styled with CSS, they can also be styled with CSS
+Custom Properties. This enables artists and designers to theme an
```

Comment:
link to CSS custom properties in either MDN or docs

NOTE: add this to required knowledge. Maybe even styling in shadow DOM

Reply:
done!